### PR TITLE
add Indonesian translation for README.md

### PR DESCRIPTION
**Pull request by schmerziel**
Complete formal translation is possible, but it will result in some weird terminologies and stuff, therefore the translation provided here is as formal as possible, but not too formal.

### CODE REVIEW

File: README.id.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+Eureka dibangun dengan pemikiran ketersediaan yang tinggi:
+
+ * Registrynya terdistribusi (local cache pada setiap client).
+ * Client  _dapat_ beroperasi tanpa server discovery (apabila servernya down).
+ * Client mengambil deltas untuk memperbarui registry.
+
+#### Dashboard
+
+Dashbord Eureka berbasis web terenabled sebagai default. Ini menunjukkan informasi-informasi penting seperti metadata servis.
```
Comment (naXa777) [inline README.id.md]: please double check this line

File: README.id.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+## Deskripsi
+
+> [Cloud-native](https://pivotal.io/de/cloud-native) merupakan pendekatan yang digunakan untuk membangun dan menjalankan aplikasi yang memanfaatkan keuntungan dari model komputasi awan.
+
+> [Microservices](https://pivotal.io/microservices) merupakan pendekatan yang digunakan untuk mengembangkan sebuah aplikasi sebagai suatu koleksi dari servis-servis kecil; setiap servis ini mengimplementasikan kapabilitas bisnis, jalan dalam proses tersendiri dan berkomunikasi melalui API HTTP. Setiap microservice dapat dideploy, upgrade, scale, dan restart namun tidak bergantung pada servis lain pada aplikasi tersebut, yang biasanya merupakan bagian dari sistem otomata.
+
+[(c) Pivotal](https://pivotal.io/de/cloud-native)
+
+Kita membagi aplikasi besar menjadi bagian-bagian yang lebih kecil yang disebut 'services'. Dan setiap dari servis ini dapat dideploy atau scale secara tersendiri._Bagaimana suatu servis mengetahui yang lainnya?_
```
Comment (naXa777) [inline README.id.md]: a whitespace is missing before `_Bagaimana`

**Comment by naXa777** [comment]
Great work! Thank you. please pay attention to my comments

File: README.id.md (hunk 1 above)
Comment (schmerziel) [reply README.id.md]: Thanks for the heads up, it's the translation that's as accurate as possible. For example, the word `default` does not have a direct translation to Indonesia. Closest thing possible is probably `biasanya/awalnya` which if translated back, is `usually/in the beginning`. I've never seen and heard someone prefer `dasbor` over `dashboard`, so it's not changing, too.

File: README.id.md (hunk 2 above)
Comment (schmerziel) [reply README.id.md]: added whitespace, thanks for this. the symbol `_` is barely visible in my laptop.

**Comment by schmerziel** [comment]
really sorry, can't keep up with the schedule. should be fine now.

**Pull request merged**